Makefile

```diff
@@ -40,6 +40,15 @@ test-coveralls: test-dependencies
 	@$(PHPUNIT) --coverage-clover build/logs/clover.xml
 	php vendor/bin/php-coveralls -v
 
+test-container:
+	@docker-compose \
+		-p icanboogie-storage-test \
+		run --rm app bash
+
+	@docker-compose \
+		-p icanboogie-storage-test \
+		down -v
+
```

Comment:
Do i need docker to run tests locally?

Reply:
Provided you have the required extensions, you can run the tests on your machine. If you don't, the Docker image provides a reproducible stable environment. If you're not familiar with Docker, I recommend you give it a try, it's quite simple.